first_run.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+echo "Installing the required libraries for the 're-id-object-tracking' module..."
+
+# Function to check if libcudnn.so.9 is present
+check_cudnn() {
+    echo "Checking if libcudnn.so.9 is installed..."
+    if ldconfig -p | grep -q 'libcudnn.so.9'; then
+        echo "libcudnn.so.9 is already installed."
+        return 0
+    else
+        echo "libcudnn.so.9 is not installed."
+        return 1
+    fi
+}
+
+# Function to install cuDNN
+install_cudnn() {
+    echo "Starting the installation of cuDNN..."
+
+    # Download the cuDNN package
+    echo "Downloading cuDNN package..."
+    wget https://developer.download.nvidia.com/compute/cudnn/9.3.0/local_installers/cudnn-local-tegra-repo-ubuntu2204-9.3.0_1.0-1_arm64.deb
+
+    # Check if the download was successful
+    if [ $? -ne 0 ]; then
+        echo "Error: Download failed. Please check your internet connection and the URL."
+        exit 1
+    else
+        echo "Download completed successfully."
+    fi
+
+    # Install the downloaded package
+    echo "Installing the downloaded cuDNN package..."
+    sudo dpkg -i cudnn-local-tegra-repo-ubuntu2204-9.3.0_1.0-1_arm64.deb
+
+    # Check if the package installation was successful
+    if [ $? -ne 0 ]; then
+        echo "Error: Failed to install the .deb package."
+        exit 1
+    else
+        echo "cuDNN package installed successfully."
+    fi
+
+    # Copy the keyring to the appropriate location
+    echo "Copying the keyring..."
+    sudo cp /var/cudnn-local-tegra-repo-ubuntu2204-9.3.0/cudnn-*-keyring.gpg /usr/share/keyrings/
```

Comment:
Discussion question: if we copy these, someone in the future won't know where they came from. Should they be symlinks? Then again, many security-focused systems refuse to follow symlinks because the other side might not be locked down properly and could be edited by bad actors (especially in `/var`). 

I dunno; maybe leave as-is.